setup.py

```diff
@@ -51,7 +51,7 @@
 
     python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*',
     install_requires=['matplotlib>=2.0.0', 'numpy>=1.11.0', 'scipy>=0.17.0',
-                      'pint>=0.8', 'xarray>=0.10.7', 'enum34;python_version<"3.4"',
+                      'pint==0.8.1', 'xarray>=0.10.7', 'enum34;python_version<"3.4"',
```

Comment:
How about: `pint>=0.8, !=0.9`? That way it will work with a future release of pint automatically?

Reply:
This assumes the future release "fixes" the issue. I'm fine with that assumption if you are.

Reply:
It does assume this. "Assuming" I can summon motivation to do anything beyond github comments and avoid falling asleep (being sick sucks) I'll be able to dig into pint a bit and open an issue. So far, I haven't seen any traffic on `pint` that makes me think a 0.9.1 is imminent.